### DISCUSSION TITLE
More information about clients

### DISCUSSION THREAD

**Discussion by anjor** [modelcontextprotocol/modelcontextprotocol#89] [Q&A - Client implementation]
### Pre-submission Checklist

- [X] I have checked that this question would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions and documentation for answers

### Question Category

- [X] Protocol Specification
- [X] SDK Usage
- [ ] Server Implementation
- [X] General Implementation
- [X] Documentation
- [ ] Other

### Your Question

There is a lot of information about how to build your own servers. Is there documentation available about how to build clients other than the existing ones (Claude desktop app, Zed, etc.)?

**Comment by AaronAbuUsama** [comment]
ive been struggling with this also, going to hack on a cli implementation today. would love you to have a look when i push 
@anjor

**Comment by dsp-ant** [comment]
I think this is valid concern. At the moment we have too little documentation available on how to create clients. See https://github.com/modelcontextprotocol/docs/issues/52, which is tracking this.

**Comment by localden** [comment]
We've limited Discussions in this repository to meeting notes and are closing this thread. To continue the conversation, pick a channel from [Contributor Communication](https://modelcontextprotocol.io/community/communication). Thanks for posting.
